docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md

```diff
@@ -148,79 +148,96 @@ helm package charts
 
 ![应用负载](png/应用负载.png)
 
-  点击 **部署新应用**，在 **来自应用模板**窗口中选择 `radondb-mysql`。
+  点击 **部署新应用**，在 **来自应用模板**窗口中选择 `RadonDB MySQL`。
 
 ![部署新应用](png/部署新应用.png)
 
   编辑应用模板基本信息。
-  
-![基本信息](png/基本信息.png)
 
+  ![基本信息](png/基本信息.png)
+  
   编辑应用配置，点击**部署** 即可完成部署。
 
-![应用配置](png/应用配置.png)
+  ![应用配置](png/应用配置.png)
 
   >说明：示例使用默认参数部署。
 
 #### **步骤 3：部署校验**
 
-在**项目管理**管理中心，选择 **应用负载 > 工作负载**，并选择**有状态副本集**页签，可查看到release 名为 `my-release` 的副本集，则 radondb-mysql 集群已成功部署。
+在**项目管理**管理中心，选择 **应用负载 > 工作负载**，并选择**有状态副本集**页签，可查看到 release 名为 `demo` 的副本集，则 RadonDB MySQL 集群已成功部署。
```

Comment:
The name of the statefulset is `demo-radondb-mysql`
https://github.com/runkecheng/radondb-mysql-kubernetes/blob/rename_keyword/docs/KubeSphere/png/%E6%8E%A7%E5%88%B6%E5%8F%B0%E9%83%A8%E7%BD%B2%E6%88%90%E5%8A%9F.png